conf/routes

```diff
@@ -39,6 +39,7 @@ GET  /monthly-contributions                         controllers.Application.cont
 GET  /contribute                                    controllers.Application.contributeGeoRedirect()
 GET  /$country<(uk|us|au|eu|int|nz|ca)>/contribute  controllers.Application.contributionsLanding(country: String)
 GET  /contribute.new                                controllers.Application.newContributionsLanding()
+GET  /thankyou.new                                  controllers.Application.newThankyouLanding()
```

Comment:
Happy to discuss. I expect these two routes to go away very soon. In fact, it may be as early as tomorrow, as once this PR goes out I'll do the React transition right away.